taskcluster/ci/beetmover/kind.yml

```diff
@@ -41,7 +42,9 @@ tasks:
   nimbus:
     description: 'Publish {} to archive.mozilla.org'
     from-deps:
-      kinds: [nimbus-binaries-assemble]
+      kinds:
+         - nimbus-binaries-assemble
+         - swift
```

Comment:
Just want to double check that the intended behaviour here is to have one beetmover task per each `swift` and each `nimbus-binaries-assemble` task.

Reply:
That seems right to me.